lib/utf8_sanitizer.rb

```diff
@@ -22,7 +24,23 @@ def call(env)
 
   private
 
-  def invalid_strings(values)
+  # @param [Hash] params
+  def params_have_null_byte?(params)
+    params.values.any? { |value| has_null_byte?(value) }
+  end
+
+  def has_null_byte?(param)
+    case param
+    when Hash
+      param.values.any? { |value| has_null_byte?(value) }
+    when Array
+      param.any? { |value| has_null_byte?(value) }
+    when String
+      param.include?("\x00")
+    end
+  end
+
+  def has_invalid_strings(values)
```

Comment:
Just checking my understanding of naming conventions here. Would we want to drop the `has_` or add a `?` with this?

```suggestion
  def invalid_strings?(values)
```

Reply:
I feel more strongly about `is_` instead of `has_` but you're right, the Rubocop rule for predicate methods would demand be renamed, if we had it enabled: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/PredicateName

Reply:
Updated in 66298e6330760e86aae9071b0316506795d66759